lib/questions.rb

```diff
@@ -63,33 +80,44 @@ def separate_array_into_even_and_odd_numbers(array)
 # e.g. 'bob'. So in the array ['bob', 'radar', 'eat'], there
 # are 2 palindromes (bob and radar), so the method should return 2
 def number_of_elements_that_are_palindromes(array)
+  array.delete_if {|word| word != word.reverse}
+  array.length
 end
 
 # return the shortest word in an array
 def shortest_word_in_array(array)
+  array.min{|a,b| a.size <=> b.size }
```

Comment:
Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.